configurations/batch/batch.ini.template

```diff
@@ -800,6 +801,12 @@ scriptPath                                          = batches/ServerNodeMonitor/
 params.serverNodeTTL                                = 120
 params.typesToMonitor                               = 1
 
+[KAsyncLiveEntryArchive : JobHandlerWorker]
+id                                                  = 680
```

Comment:
Please note this id is already taken by KAsyncCopyCaptions
[KAsyncCopyCaptions : JobHandlerWorker]
id                                    = 680

Please double check yourself before attaching an id to a worker